devicesetup/src/main/java/io/particle/android/sdk/accountsetup/LoginActivity.java

```diff
@@ -224,10 +224,7 @@ private boolean isEmailValid(String email) {
     }
 
     private boolean isPasswordValid(String password) {
-        return (password.length() > 0);
+        return (password.length() > 7);
```

Comment:
It should not be checked for Login only for signup because otherwise users who previously set passwords <8 chars will not be able to login